src/app/plan/components/Wizard/VolumesForm.tsx

```diff
@@ -26,29 +27,33 @@ class VolumesForm extends React.Component<any> {
     this.props.onWizardLoadingToggle(true);
   }
 
-  componentDidUpdate() {
-    const { plans, values } = this.props;
+  componentDidUpdate(prevProps) {
+    if (prevProps !== this.props) {
+      const { plans, values } = this.props;
 
-    if(plans.length === 0) {
-      return;
-    }
+      if (plans.length === 0) {
+        return;
+      }
 
-    const currentPlan = plans.find(p => {
-      return p.metadata.name === values.planName;
-    });
+      const currentPlan = plans.find(p => {
+        return p.metadata.name === values.planName;
+      });
 
-    if(!currentPlan.status) {
-      return;
-    }
+      if (currentPlan) {
```

Comment:
Fixed a bug causing the app to crash when state was between renders & currentPlan was not defined yet